examples/energy-management-app/energy-management-common/energy-management-app.zap

```diff
@@ -746,6 +746,266 @@
             }
           ]
         },
+        {
+          "name": "Localization Configuration",
```

Comment:
were these added manually, or was this a zap addition? ie, do you WANT these clusters, or is this a weird zap side effect?